caster-editor/src/components/DialogRenameNode.vue

```diff
@@ -13,8 +15,8 @@
         <span class="dialog-footer">
           <ElButton @click="emit('cancel')">Cancel</ElButton>
           <ElButton
-            color="#ADFF00"
             type="primary"
+            color="#ADFF00"
```

Comment:
I think we should avoid hardcoding colors, and in this case we use the `primary` button which uses the `primary` color, so we should overwrite the css variable `primary-color` globally.

Reply:
Yes, deleted it. It's already covered by the css

Reply:
In this case its approved :) Ready for merge!